cluster-scope/base/user.openshift.io/groups/operate-first/group.yaml

```diff
@@ -1,6 +1,19 @@
----
 apiVersion: user.openshift.io/v1
 kind: Group
 metadata:
   name: operate-first
-users: []
+users:
+  - anishasthana
+  - accorvin
+  - 4n4nd
+  - humairak
+  - hemajv
+  - durandom
+  - martinpovolny
```

Comment:
martin shouldn't be on this group now right?

Reply:
We're not doing a cleanup yet. Right now we should aim for 1 to 1 mapping of current RBAC. Once we have every group in plaintext we can start with a cleanup. :slightly_smiling_face: 

I'll create additional issue for RBAC revisiting and aggregate your comment in there, sounds good? :+1:

Reply:
https://github.com/operate-first/support/issues/357

Reply:
Sounds good! Thanks